flag_engine/segments/models.py

```diff
@@ -1,15 +1,10 @@
-import re
 import typing
-from contextlib import suppress
 
-import semver
 from pydantic import BaseModel, Field
 
 from flag_engine.features.models import FeatureStateModel
 from flag_engine.segments import constants
 from flag_engine.segments.types import ConditionOperator, RuleType
-from flag_engine.utils.semver import is_semver
-from flag_engine.utils.types import get_casting_function
 
 
 class SegmentConditionModel(BaseModel):
```

Comment:
`_EXCEPTION_OPERATOR_METHODS` can be removed now, right?

Reply:
Good call. Done.